agent/dns_test.go

```diff
@@ -2913,7 +2858,7 @@ func TestDNS_ServiceLookup_AnswerLimits(t *testing.T) {
 		expectedANYQuery    int
 		expectedANYQueryID  int
 	}{
-		{"0", 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+		//{"0", 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
```

Comment:
Curious what happened here? Does this fail to validate then we can just delete it.

Reply:
The new code did not allow `dns_config.udp_answer_limit` to be zero which made this test fail. I guess we commented this out while getting things to work. I've restored the previous behavior.